packages/shared/components/inputs/Password.svelte

```diff
@@ -58,6 +58,7 @@
             {submitHandler}
             disableContextMenu={true}
             spellcheck="false"
+            autocomplete="false"
```

Comment:
Yikes, TIL Chrome just doesn't support `autocomplete="off"` even though that's what MDN says to do :sweat_smile:

Reply:
yes and seems that the autocomplete is showed when the user is opting to reveal the password 🤔